scripts/generate-usage-data/generate-usage-GB-Ingested.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime, timedelta, timezone
+from pprint import pprint
+import requests, argparse, os
+from dateutil.parser import parse
+import random
+import uuid
+import json
+import requests
+from dotenv import load_dotenv
+load_dotenv()
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--customer_id', type=str, required=True)
+parser.add_argument('--days_of_data', type=int, required=False, default=33)
+
+token = os.getenv('API_TOKEN')
+if not token:
+    raise ValueError("Set API_TOKEN environment variable to metronome API key")
+
+args = parser.parse_args()
+customer_id = args.customer_id
+if not customer_id:
+    raise ValueError("customer_id must be set")
+
+days_of_data = args.days_of_data
+if days_of_data > 33 or days_of_data < 1:
+    raise ValueError("days_of_data needs to be greater than 1 and less than 34")
+
+
+def ingest(events):
+    remainder = events
+    while len(remainder) > 0:
+        pprint(remainder[:100])
+
+        response = requests.post(
+            "https://api.metronome.com/v1/ingest",
+            headers={
+                "Authorization": f"Bearer {token}",
+            },
+            json=remainder[:100]
+        )
+        print(response.status_code)
```

Comment:
lets not print this response code since there will be quite a few

Reply:
resolved